client/apis/volumegroupsnapshot/v1alpha1/types.go

```diff
@@ -0,0 +1,328 @@
+
+	// DeletionPolicy determines whether this VolumeGroupSnapshotContent and the
+	// physical group snapshot on the underlying storage system should be deleted
+	// when the bound VolumeGroupSnapshot is deleted.
+	// Supported values are "Retain" and "Delete".
+	// "Retain" means that the VolumeGroupSnapshotContent and its physical group
+	// snapshot on underlying storage system are kept.
+	// "Delete" means that the VolumeGroupSnapshotContent and its physical group
+	// snapshot on underlying storage system are deleted.
+	// For dynamically provisioned group snapshots, this field will automatically
+	// be filled in by the CSI snapshotter sidecar with the "DeletionPolicy" field
+	// defined in the corresponding VolumeGroupSnapshotClass.
+	// For pre-existing snapshots, users MUST specify this field when creating the
+	// VolumeGroupSnapshotContent object.
+	// Required.
+	DeletionPolicy snapshotv1.DeletionPolicy `json:"deletionPolicy" protobuf:"bytes,2,opt,name=deletionPolicy"`
+
+	// Driver is the name of the CSI driver used to create the physical group snapshot on
+	// the underlying storage system.
+	// This MUST be the same as the name returned by the CSI GetPluginName() call for
+	// that driver.
+	// Required.
+	Driver string `json:"driver" protobuf:"bytes,3,opt,name=driver"`
+
+	// VolumeGroupSnapshotClassName is the name of the VolumeGroupSnapshotClass from
+	// which this group snapshot was (or will be) created.
+	// Note that after provisioning, the VolumeGroupSnapshotClass may be deleted or
+	// recreated with different set of values, and as such, should not be referenced
+	// post-snapshot creation.
+	// For dynamic provisioning, this field must be set.
+	// This field may be unset for pre-provisioned snapshots.
+	// +optional
+	VolumeGroupSnapshotClassName *string `json:"volumeGroupSnapshotClassName,omitempty" protobuf:"bytes,4,opt,name=volumeGroupSnapshotClassName"`
+
+	// Source specifies whether the snapshot is (or should be) dynamically provisioned
+	// or already exists, and just requires a Kubernetes object representation.
+	// This field is immutable after creation.
+	// Required.
+	Source VolumeGroupSnapshotContentSource `json:"source" protobuf:"bytes,5,opt,name=source"`
+}
+
+// VolumeGroupSnapshotContentStatus defines the observed state of VolumeGroupSnapshotContent.
+type VolumeGroupSnapshotContentStatus struct {
+	// VolumeGroupSnapshotHandle is a unique id returned by the CSI driver
+	// to identify the VolumeGroupSnapshot on the storage system.
+	// If a storage system does not provide such an id, the
+	// CSI driver can choose to return the VolumeGroupSnapshot name.
+	// +optional
+	VolumeGroupSnapshotHandle *string `json:"volumeGroupSnapshotHandle,omitempty" protobuf:"bytes,1,opt,name=volumeGroupSnapshotHandle"`
+
+	// CreationTime is the timestamp when the point-in-time group snapshot is taken
+	// by the underlying storage system.
+	// If not specified, it indicates the creation time is unknown.
```

Comment:
Add this:	
        // The format of this field is a Unix nanoseconds time encoded as an int64.
	// On Unix, the command `date +%s%N` returns the current time in nanoseconds
	// since 1970-01-01 00:00:00 UTC.